docs/source/initialization.md

```diff
@@ -83,10 +85,14 @@ If you wish to make your own `InterceptorProvider` instead of using the provided
 
 - `ResponseCodeInterceptor` checks to make sure a valid response status code has been returned. **NOTE**: Most errors at the GraphQL level are returned with a `200` status code and information in the `errors` array per the GraphQL Spec. This interceptor helps with things like server errors and errors that are returned by middleware. [This article on error handling in GraphQL](https://medium.com/@sachee/200-ok-error-handling-in-graphql-7ec869aec9bc) is a really helpful look at how and why these differences occur. 
 - `AutomaticPersistedQueryInterceptor` handles checking responses to see if an error is because an automatic persisted query failed, and the full operation needs to be resent to the server.
-- `LegacyParsingInterceptor` parses code generated by our Typescript code generation. 
-- `LegacyCacheWriteInterceptor` writes to a provided `ApolloStore`. 
+- `LegacyParsingInterceptor` parses code generated by our current Typescript-based code generation. 
```

Comment:
Why not to rename it to `TypescriptGeneratedParsingInterceptor` - ok name is not best but at least no-one will run to look for non legacy interceptor